packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -54,7 +54,7 @@ export class Tasks extends React.Component {
               const TaskComponent = getTaskComponent(task.type)
               if (TaskComponent) {
                 return (
-                  <Box key={task.taskKey} basis='246px'>
+                  <Box key={task.taskKey} basis='auto'>
```

Comment:
I think the fixed size was to make it so that the box div had a min height. But it looks like this is causing issues, so we'll need to do this another way. Switching to auto is best for now.